### PR TITLE
Image selection

### PR DESCRIPTION
**Pull request by koyaan**
Add img-selection module

### CODE REVIEW

**Comment by koyaan** [comment]
see https://github.com/mms-imagebrowser/image-browser/issues/6

File: src/app/img-selection/img-selection.component.ts [hunk 1]
```diff
@@ -7,9 +10,44 @@ import { Component, OnInit } from '@angular/core';
...
+        response.children.forEach(file => {
+          if(!file.children) {
+            this.images.push(new ImageFile(file.name, file.path, file.extension, false, file.size));
+          }
+        });
+      });
+    });
+
+    // Subscription PoC
+    this.selectionService.getImages().subscribe(selectedImages => {
```
Comment (mik3ydll) [inline src/app/img-selection/img-selection.component.ts:44]: just for testing purpose?

**Review by mik3ydll — APPROVED** [review]
looks fine
maybe you could still move the file serving to the express server as described here: https://expressjs.com/en/starter/static-files.html

File: src/app/img-selection/img-selection.component.ts (hunk 1 above)
Comment (koyaan) [reply src/app/img-selection/img-selection.component.ts:44]: yes just to test the subscription

**Comment by koyaan** [comment]
images are now served by express

**Pull request merged**